discordia/containers/Embed.lua

```diff
@@ -0,0 +1,35 @@
+local Snowflake = require('./Snowflake')
+
+local format = string.format
+
+local Embed, property, method = class('Embed', Snowflake)
+Embed.__description = "Represents Discord Embed."
+
+function Embed:__init(data, parent)
```

Comment:
Why are you manually copying values? This is done automatically when `Container` is constructed.